src/future-hopr-lib-components/Layout/drawer.tsx

```diff
@@ -23,8 +23,8 @@ const StyledDrawer = styled(MuiDrawer)`
     padding-top: 59px;
     transition: width 0.4s ease-out;
     overflow-x: hidden;
-    background: #3c64a5;
-    color: white;
+    /* background: #3c64a5;
+    color: white; */
```

Comment:
we should remove the comments on the styled components.

Reply:
Left it as I wanted to re-use it in next pr for the Staking Hub drawer. It's fixed now